src/Host/Storage.php

```diff
@@ -14,6 +14,73 @@
 
 class Storage
 {
+	/**
+	 * In a multi user environment where multiple projects are located on the same instance(s)
+	 *	if two or more users are deploying at the same time on the same environment, 
+	 *	the generated hosts files are overwritten or worse, the deploy will fail 
+	 *	due to file permissions
+	 *
+	 * Make the persistent storage folder configurable or use the system tmp 
+	 *	as default if not possible
+	 * 
+	 * @return string
+	 * @throws Exception
+	 */
+	private static function _getPersistentStorageLocation() {
```

Comment:
Rename `_getPersistentStorageLocation` to `getPersistentStorageLocation` and move this method to end of class.